crates/primitives/starknet/src/transaction/utils.rs

```diff
@@ -0,0 +1,29 @@
+use alloc::collections::BTreeMap;
+use alloc::vec::Vec;
+
+use frame_support::BoundedVec;
+#[cfg(feature = "std")]
+use starknet_core::types::{LegacyContractEntryPoint, LegacyEntryPointsByType};
+
+use crate::execution::types::{EntryPointTypeWrapper, EntryPointWrapper, MaxEntryPoints};
+
+/// Returns a btree map of entry point types to entrypoint from deprecated entry point by type
+#[cfg(feature = "std")]
```

Comment:
why did you move that ?

Reply:
because it's used in `to_declare_tx` and since we are moving that logic to a `try_from` implementation in a primitive, I thing to bring that function over as well. Correct me if I'm wrong

Reply:
makes sense. But make sure that you rebased, this function doesn't exist anymore, it has been changed to use hashmaps